python/grass/jupyter/interact_display.py

```diff
@@ -0,0 +1,228 @@
+#
+# AUTHOR(S): Caitlin Haedrich <caitlin DOT haedrich AT gmail>
+#
+# PURPOSE:   This module contains functions for interactive display
+#            in Jupyter Notebooks.
+#
+# COPYRIGHT: (C) 2021 Caitlin Haedrich, and by the GRASS Development Team
+#
+#            This program is free software under the GNU Gernal Public
+#            License (>=v2). Read teh file COPYING that comes with GRASS
+#            for details.
+
+import os
+import sys
+import subprocess
+import folium
+import grass.script as gs
+import shutil
+import codecs
+
+def create_location(
+    dbase,
+    location,
+    epsg=None,
+    overwrite=False,
+    env=None
+):
+    """Create new location
+    Raise ScriptError on error.
+    :param str dbase: path to GRASS database
+    :param str location: location name to create
+    :param epsg: if given create new location based on EPSG code
+    :param bool overwrite: True to overwrite location if exists(WARNING:
+                           ALL DATA from existing location ARE DELETED!)
+    """
+
+    # check if location already exists
+    if os.path.exists(os.path.join(dbase, location)):
+        if not overwrite:
+            print(("Location <%s> already exists. Operation canceled.") % location)
+            try:
+                os.remove(tmp_gisrc)
+            except Exception:
+                pass
+            return
+        else:
+            print(("Location <%s> already exists and will be overwritten") % location)
+            shutil.rmtree(os.path.join(dbase, location))
+
+    stdin = None
+    kwargs = dict()
+
+    ps = gs.pipe_command(
+        "g.proj",
+        quiet=True,
+        flags="t",
+        epsg=epsg,
+        location=location,
+        stderr=subprocess.PIPE,
+        env=env,
+        **kwargs,
+    )
+    
+    error = ps.communicate(stdin)[1]
+    try:
+        os.remove(tmp_gisrc)
+    except Exception:
+        pass
+
+    if ps.returncode != 0 and error:
+        raise ScriptError(repr(error))
+
+    try:
+        fd = codecs.open(
+            os.path.join(dbase, location, "PERMANENT", "MYNAME"),
+            encoding="utf-8",
+            mode="w",
+        )
+        
+        fd.write(os.linesep)
+        fd.close()
+    except OSError as e:
+        raise ScriptError(repr(e))
+
+class GrassRendererInteractive():
```

Comment:
I still think there might be a better, completely different, name, but in any case, the adjectives should go first, noun last, e.g., GrassInteractiveRenderer.

Reply:
What about InteractiveMap? The idea that is for GRASS is already implied since the library is called grass.jupyter.